mongodbatlas/resource_mongodbatlas_federated_database_instance_migration_test.go

```diff
@@ -0,0 +1,156 @@
+package mongodbatlas
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+)
+
+func TestAccFederatedDatabaseInstance_MigraitonTest_basic(t *testing.T) {
```

Comment:
Could be worth leaving a comment adding context for the definition of these migration tests, because in this case it is not related to the framework migration but for migrating the SDK.

Reply:
@AgustinBettati True, but essentially we can be using the same tests for both migrations. This would help us to identify any breaking changes we might be introducing as part of either migration.

We can always extend the same tests if required during migration to the Framework.

Reply:
Makes sense 👍